### PR TITLE
more

### PR DESCRIPTION
**Pull request by grosser**
@adammw 

I'll just pile fixes/changes here until you get a chance to review :)

### CODE REVIEW

**Comment by grosser** [comment]
oh yeah `coverage: 100.0% of statements` :D

File: README.md [hunk 1]
```diff
@@ -1,20 +1,34 @@
-# kinesiscat
+# kinesiscat [![Build Status](https://travis-ci.com/adammw/kinesiscat.png)](https://travis-ci.com/adammw/kinesiscat) [![Coverage](https://img.shields.io/badge/Coverage-100%25-green.svg)](https://github.com/grosser/go-testcov)
```
Comment (adammw) [inline README.md:1]: any services to auto-calculate the code coverage? don't really want to rely on updating the readme each time it changes

File: pkg.go [hunk 2]
```diff
@@ -150,14 +159,31 @@ func parallel(things []string, fn func(string)) {
...
 }
 
+func showAvailableStreams(client kinesisiface.KinesisAPI) {
+	out, err := client.ListStreams(&kinesis.ListStreamsInput{})
+	fatalfIfErr("get streams error: %v", err)
+	fmt.Fprintf(os.Stderr, "No stream name give, please chose one of these streams:\n")
+	for _, name := range out.StreamNames {
+		fmt.Fprintf(os.Stderr, "%v\n", *name)
+	}
+	exitFn(2)
```
Comment (adammw) [inline pkg.go:169]: nit: would prefer the exit call in main()

**Comment by grosser** [comment]
maybe change to `good enough` instead of `100` or so :D

**Pull request merged**